README.md

```diff
@@ -21,7 +21,11 @@ Before you can start working with Yari, you need to:
 <!-- markdownlint-disable list-marker-space -->
 
 1.  Install [git](https://git-scm.com/), [Node.js](https://nodejs.org), and
-    [Yarn 1](https://classic.yarnpkg.com/en/docs/install).
+    [Yarn 1](https://classic.yarnpkg.com/en/docs/install). Alternatively, if you
+    have [Nix](https://nixos.org) installed on your system, you can use the dev
+    shell provided in the `flake.nix` file at the root of this repo with
+    `nix develop`, to quickly setup an environment with all of the required
```

Comment:
```suggestion
    `nix develop`, to quickly set up an environment with all of the required
```
…because “set up” for the (compound) verb, while “setup” is a noun